installation/resources/installer-config-gke-benchmark.yaml.tpl

```diff
@@ -9,7 +9,7 @@ metadata:
     kyma-project.io/installation: ""
 data:
   global.externalServicesMock.enabled: "true"
-  global.externalServicesMock.auditlog: "false"
+  global.externalServicesMock.auditlog.applyMockConfiguration: "false"
```

Comment:
Shouldn't this be set to true?

Reply:
The auditlog e2e tests were disabled for benchmark jobs even before this PR, so this configuration is really obsolete and doesn't have an effect on the end result